examples/src/main/java/io/dapr/examples/bindings/http/OutputBindingExample.java

```diff
@@ -25,7 +24,7 @@ public MyClass(){}
   }
 
   public static void main(String[] args) throws Exception {
-    DaprClient client = new DaprClientBuilder().build();
+    DaprClient client = new DaprClientBuilder(null).build();

```

Comment:
This seems like it would be more natural if there was an overload that took in no args to represent this.

Reply:
I agree. On the other hand, the feedback we got is to make it a required argument, so users must pass in something and be aware that they are OK with the default behavior.

Reply:
Passing null into the constructor of a builder makes the attribute not required, having a withSerializer method to add the serializer will be more intuitive for users of the API, regarding what´s expected, By defualt a Builders builds base on default behaviors, only if you want to change something you send it before building, but only Required attributes that do not have a defuault behavior are passes in the constructor.

Reply:
I will make it required and handle null as exception.